data/extension/init/templates/tinygo/envoy.filters.http/default/go.mod

```diff
@@ -0,0 +1,8 @@
+module envoy.filters.http
```

Comment:
hm, `envoy.filters.http` is a valid module name ?

In that case, let's change it to something like `my.http_filter`